### DISCUSSION TITLE
Comment calculer ce que l'on doit ?

### DISCUSSION THREAD

**Discussion by rodolphebejot** [betagouv/mon-entreprise#2885] [Q&A] [answered]
Bonjour,
En utilisant le package modele-social il semblerait que la déclaration du chiffres d'affaires pour auto entreprenneur se fasse à l'année. 
Pourtant il semble que l'on puisse payer l'URSSAF soit trimestriellement ou bien mensuellement

Est ce que je rate qqc ?

Merci pour votre aide

**Comment by rodolphebejot** [comment]
S'il vous plaît 😢

**Comment by johangirod** [comment, marked as answer]
Bonjour ! 
Pour l'instant, il n'est pas possible d'effectuer un calcul au trimestre avec le paquet `modele-social`. En revanche vous pouvez : 
- Effectuer le calcul par an et diviser par quatre le résultat
- Effectuer le calcul par mois et multiplier par trois le résultat

Sachant que si l'ACRE est activée, elle sera prise en compte au prorata de l'année en cours (et non du trimestre en cours).
